src/pybind/mgr/orchestrator.py

```diff
@@ -670,7 +670,8 @@ class PlacementSpec(object):
     def __init__(self, label=None, nodes=[]):
         self.label = label
 
-        self.nodes = [parse_host_specs(x, require_network=False) for x in nodes]
+        self.nodes = [parse_host_specs(x, require_network=False) for x in nodes if x]
+
```

Comment:
remove this empty line please. (PEP8)

Reply:
PEP8 expects 2 blank lines after a class definition. The rest of the file is structured the same way.